vulcanai2/models/basenetwork.py

```diff
@@ -132,6 +122,16 @@ def __init__(self, name, config, in_dim=None, save_path=None,
             validation_accuracy=[]
         )
 
+        if in_dim is not None:
```

Comment:
would be better to be `if in_dim:`, no? what if they give an empty list?

Reply:
yup will fix